publish.html

```diff
@@ -552,7 +552,7 @@ <h2><a class="anchor" id="repository"></a>Using OMERO as a Public Repository</h2
 
 <li><p><a href="http://jcb-dataviewer.rupress.org" target="_blank" class="external-url">Journal of Cell Biology (JCB) Data Viewer</a></p></li>
 
-<li><p><a href="http://odr.stowers.org/websimr/" target="_blank" class="external-url">Stowers Institute Original Data Repository (ODR)</a></p></li>
+<li><p><a href="https://odr.stowers.org/websimr/" target="_blank" class="external-url">Stowers Institute Original Data Repository (ODR)</a></p></li>
```

Comment:
At the moment this hangs for me; the http works fine. (Again, should link checking have revealed this?)